packages/notifications/src/styled/StyledBase.ts

```diff
@@ -6,40 +6,76 @@
  */
 
 import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
-import { getColorV8, getLineHeight, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import {
+  getLineHeight,
+  DEFAULT_THEME,
+  getColor,
+  retrieveComponentStyles
+} from '@zendeskgarden/react-theming';
 import { Type } from '../types';
+import { validationTypes } from '../utils/icons';
 
-export interface IStyledBaseProps {
-  isFloating?: boolean;
-  hue?: string;
-  type?: Type;
+const COMPONENT_ID = 'notifications.base_container';
+
+export interface IStyledBaseProps extends ThemeProps<DefaultTheme> {
+  $isFloating?: boolean;
+  $type?: Type;
 }
 
-const boxShadow = (props: ThemeProps<DefaultTheme>) => {
+const boxShadow = (props: IStyledBaseProps) => {
```

Comment:
Recommend getting rid of the `boxShadow` function .. per [API docs](https://github.com/zendeskgarden/react-components/blob/main/docs/api.md#rules), this should just be part of `colorStyles`.